app/controllers/timelogs_controller.rb

```diff
@@ -0,0 +1,61 @@
+class TimelogsController < ApplicationController
+  before_action :load_user
+
+  def index
+    @timelog = Timelog.where(user_id: current_user.id)
```

Comment:
it is better to do `@timelogs = current_user.timelogs`.
Also, always use plural form for variables if it has to be plural (in our case, timelogs, not timelog).